gridcapa-task-manager-app/src/main/java/com/rte_france/farao/gridcapa_task_manager_app/TaskManagerController.java

```diff
@@ -31,4 +39,11 @@ public TaskManagerController(TaskManager taskManager) {
     public ResponseEntity<TaskDto> getTaskFromTimestamp(@PathVariable String timestamp) {
         return ResponseEntity.ok().body(taskManager.getTaskDto(LocalDateTime.parse(timestamp)));
     }
+
+    @PostMapping(value = "/tasks/launch/{timestamp}")
```

Comment:
As we have the path 'tasks/{timestamp}' already maybe 'tasks/{timestamp}/launch', I don't know, I'm not sure of the best practice here

Reply:
I am not sure myself so I did not change it for the moment. Once we get everything working I think it will be easy to change it but I don't see this as a priority